docs/android/v2/changelog/release-notes.mdx

```diff
@@ -3,6 +3,16 @@ title: Release Notes
 nav: 15
 description: Release Notes for 100ms Android SDK
 ---
+## v2.5.6 - 2023-01-11
+### Added
+* Send `requested_by` in acceptChangeRole message
+* Added `CameraControls` interface to `HMSLocalVideoTrack` - Use this interface to control the settings of native camera
+* Added Utils method to know about the properties of vp8 codecs present in the device
+
+### Fixed
+* `captureBitmap` API of the local `HMSVideoView` now returns the bitmap at the publishing resolution of local peer
+* Issue where a remote track was getting removed inside `HMSPeer` if the ordering of onRemoteTrack message from server was wrong.
```

Comment:
wrong ... or unexpected sequence of events ? :)

Reply:
right